examples/vpc_service_control/README.md

```diff
@@ -0,0 +1,129 @@
+# VPC Service Control Demo
```

Comment:
Let's change this repo to `onprem_demo`.

Reply:
Done.

Reply:
I don't see the change. I don't think you push the commit for this change.

Reply:
Done for real. :)